src/slurm_plugin/slurm_resources.py

```diff
@@ -165,13 +166,14 @@ def is_drained(self):
 
     def is_powering_down(self):
         """Check if slurm node is in powering down state."""
-        return any(
-            powering_down_state in self.states for powering_down_state in self.SLURM_SCONTROL_POWERING_DOWN_STATES
+        return (
+            any(powering_down_state in self.states for powering_down_state in self.SLURM_SCONTROL_POWERING_DOWN_STATES)
+            and self.SLURM_SCONTROL_POWERED_STATE not in self.states
         )
```

Comment:
Don't we need to assume that a node in `IDLE+CLOUD+POWER_DOWN+POWERED_DOWN` is actually powering down?

Reply:
After our discussion, treat `POWERING_DOWN + POWERED_DOWN` as powering down node